### ISSUE TITLE
Uploading metadata - easy question

### ISSUE THREAD

**Issue by mcgeo52** [vimeo/vimeo.php#218]
The documentation says:
"If you wish to add metadata to your video as it's being uploaded, you can supply a data object to .upload()."

I am currently uploading the name and description with no problem. The api returns the uri exactly like it should.

However, when I play the video it shows three other videos at the end. I would really like to get rid of the related videos.

I have seen how I can do it going into each video individually on Vimeo, but I'm looking at hundreds of videos for my first few test cases. Given that the tests go as expected, I could be looking at hundreds of thousands of videos.

In short, going in manually to it is not an option. If I'm reading the documentation correctly, I could do it through the api after the videos are loaded, but that seems like a real waste of time if there is some way I can include the code in the upload.

Is there a simple variable I can set in the metadata for the upload to prevent the related videos appearing?
